db/driver/sqlc/schema.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TYPE rate_limit_capacity_period AS ENUM ('daily', 'weekly', 'monthly');
```

Comment:
Why did you go with an enum for capacity but not throughput (e.g. per second, per minute, per hour)>

Trying to understand the thought process here

Reply:
I think throughput being defined by the second is the standard. Just my opinion here and I'm open to be wrong on this but I doubt anybody is going to want to be able to enforce throughput by units other than second. IMO no need to get that granular with it.

Whereas I see capacity as having more variability; people may want to bill/meter daily, weekly, monthly. For example we've now done both daily and monthly in our business.

Reply:
> IMO no need to get that granular with it.

That's fine, just the type of thing worth calling out in comments as to why this decision was made.